.github/workflows/automated_release.yml

```diff
@@ -0,0 +1,55 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - "v*" # Push events to matching v*, i.e. v1.0, v20.15.10
+
+name: Automated Release
```

Comment:
Nit: "Automated" seems a little redundant to me, what do you think of `release.yml`?

Reply:
Na, I think automated works fine

Reply:
Sommelier calls it release, Gravity Bridge calls it automated release, either is fine.

Reply:
All Github actions are automated, it adds no new information for new developers. Either is fine, it was just a "nit".